ci_scripts/requirements.txt

```diff
@@ -1,3 +1,6 @@
+#Requirements for the release process only
```

Comment:
Is this correct guys? I made this mistake with the pdoc (ie. put it in here despite it being needed for every CI build) requirement so think its worth having some prompt here.

Reply:
I think a better description would be requirements for the CI process, as not all the scripts are related to release.

Reply:
After talk IRL decided to change the ci_scripts directory structure to make this more obvious rather than rely on comments.